.travis.yml

```diff
@@ -16,4 +16,5 @@ deploy:
   app: qc-engine
   on:
     repo: Quick-Command/qc-engine
+    branch: main 
```

Comment:
Thanks for the fix here.  Hopefully this alleviates the issue we were seeing on Travis CI despite the fact that it was still deploying.